docs/manual/layout/site-structure/configure-pages.de.md

```diff
@@ -77,6 +73,129 @@ einer eindeutigen Beschreibung zu versehen.
+
+**Domainname:** Wenn du möchtest, dass eine Webseite in deiner Seitenstruktur unter einer bestimmten Domain wie z. B. 
+»firma.de« erreichbar ist, kannst du diese hier eingeben. Ruft ein Besucher dann »firma.de« in seinem Browser auf, wird 
+er automatisch zu dem entsprechenden Startpunkt einer Webseite weitergeleitet.
+
+**Protokoll (HTTPS verwenden):** Wenn deine Webseite über HTTPS verfügbar ist, muss diese Einstellung entsprechend
+konfiguriert werden. Ab Contao **4.10** heißt diese Einstellung **Protokoll** und lässt die Auswahl zwischen `http://`
+und `https://` zu. In Contao **4.9** werden Besucher automatisch auf HTTPS weitergeleitet, wenn diese Einstellung aktiv
+ist. Ab Contao **4.10** wird automatisch entweder zu `http://` oder `https://` weitergeleitet.
+
+{{% notice warning %}}
+Falls du für deine Domain ein SSL-Zertifikat einsetzt, dann muss diese Einstellung in Contao **4.10** und höher von `http://` 
+auf `https://` geändert. Andernfalls könnte eine unendliche Weiterleitung im Frontend passieren, falls beispielsweise die
+Hosting Umgebung automatisch von `http://` auf `https://` weiterleitet.
+{{% /notice %}}
+
+{{< version "4.10" >}}
+
+**URL-Präfix:** Mit dieser Einstellung kann ein optionaler URL-Präfix allen Seitenaliasen unterhalb dieses Startpunkts
+zugewiesen werden. Diese Einstellung ist wie erwähnt ab Contao **4.10** verfügbar, davor gab es nur über die
+`contao.prepend_locale` Einstellung die Möglichkeit einen über die Sprache definierten Präfix zu verwenden. Nun ist dieser
+Präfix frei wählbar und damit unabhängig von der jeweils eingestellten Sprache.
+
+{{< version "4.10" >}}
+
+**URL-Suffix:** Mit dieser Einstellung kann der »URL-Suffix« geändert oder entfernt werden. Der URL-Suffix wird bei der
+Generierung der URL einer Seite an den Seitenalias angehängt.
+
+{{< version "4.5" >}}
+
+**Gültige Alias-Zeichen:** Der Slug-Generator ermöglicht es einen individuellen Zeichensatz für automatisch erstellte 
+Aliase auszuwählen.
+
+| Alias-Einstellungen                  | Erklärung                                                            |
+|:-------------------------------------|:---------------------------------------------------------------------|
+| Unicode-Zahlen und -Kleinbuchstaben  | Aus dem Seitennamen »Über uns« wird das Alias `über-uns` generiert.  |
+| Unicode-Zahlen und -Buchstaben       | Aus dem Seitennamen »Über uns« wird das Alias `Über-uns` generiert.  |
+| ASCII-Zahlen und -Kleinbuchstaben    | Aus dem Seitennamen »Über uns« wird das Alias `ueber-uns` generiert. |
+| ASCII-Zahlen und -Buchstaben         | Aus dem Seitennamen »Über uns« wird das Alias `Ueber-uns` generiert. |
+
+Für die Erzeugung des Aliases ist in Einzelfällen auch die eingestellte Sprache relevant. So wird ein Deutsches »Über« 
+zu »ueber« jedoch ein Finnisches »eläinkö« zu »elainko« konvertiert.
+
+{{% notice note %}}
+Diese Einstellung befindet sich in Contao **4.5** bis **4.9** in der Sektion _Alias-Einstellungen_.
```

Comment:
```suggestion
Diese Einstellung befindet sich bis Contao **4.9** in der Sektion _Alias-Einstellungen_.
```

Reply:
In Contao 4.4 gibt es diese Einstellung jedoch nicht, daher hatte ich **4.5.** bis **4.9** geschrieben. Soll ich's trotzdem auf _bis Contao **4.9**_ ändern?

Reply:
Nein, dann ist deine Version natürlich richtig.